appconfiguration/spring-cloud-azure-starter-appconfiguration/appconfiguration-sample/README.md

```diff
@@ -0,0 +1,226 @@
+
+There are two main ways to build a Spring Boot native application.
+
+#### Run with Buildpacks
+
+- System Requirements
+
+Docker should be installed, see [System Requirements](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#getting-started-buildpacks-system-requirements) for more details.
+
+- Build the native application
+
+Run `mvn -Pbuildpack package spring-boot:build-image`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application) for more details.
+
+```shell
+mvn -Pbuildpack package spring-boot:build-image
+```
+
+- Run the native application
+
+Run `docker run --rm -p 8080:8080 appconfiguration-sample:1.0.0`, see [Run the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_run_the_native_application) for more details.
+```shell
+docker run --rm -p 8080:8080 appconfiguration-sample:1.0.0
+```
+
+#### Run with Native Build Tools
+
+- System Requirements
+
+GraalVM `native-image` compiler should be installed, see [System Requirements](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#getting-started-native-image-system-requirements) for more details. If using the Windows platform, you need to install `Visual Studio Build Tools`.
+
+- Build the native application
+
+Run `mvn -Pnative -DskipTests package`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application_2) for more details.
+
+```shell
+mvn -Pnative -DskipTests package
+```
+
+When you encounter the `The command line is too long` exception when executing the native-image command, please use **shaded** profile instead of **native** to build, and you must turn off the Spring Cloud Azure compatibility verification function (`spring.cloud.azure.compatibility-verifier.enabled=false`).
+
+```shell
+mvn -Pshaded -DskipTests package
+```
+
+- Run the native application
+
+Run `target\appconfiguration-sample`, see [Run the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_run_the_native_application_2) for more details.
+```shell
+target\appconfiguration-sample
+```
+
+## Verify This Sample
+View the below results in the console.
+
+```text
+INFO 15116 --- [           main] .s.s.a.AppConfigurationSampleApplication : Returned the from Azure App Configuration: sample-key, hello from default application
+```
+
+
+## Clean Up Resources
+After running the sample, if you don't want to run the sample, remember to destroy the Azure resources you created to avoid unnecessary billing.
+
+The terraform destroy command terminates resources managed by your Terraform project.   
+To destroy the resources you created.
+
+#### Run with Bash
+
+```shell
+terraform -chdir=./terraform destroy -auto-approve
+```
+
+#### Run with Powershell, Command Prompt or Native Tools Command Prompt
```

Comment:
We should keep the same style as other file

Reply:
I discuss with @backwind1233 , it is the same with Powershell commands, so use one title to reduce the duplicate code.